app/pages/sections/catalogue-solutions/order-item/flat/declarative/manifest.json

```diff
@@ -48,23 +48,23 @@
     }
   },
   "errorMessages": {
-    "QuantityRequired": "Enter a quantity",
+    "QuantityRequired": "Enter an estimated quantity",
     "QuantityMustBeANumber": "Quantity must be a number",
-    "QuantityInvalid": "Quantity must be a whole number",
-    "QuantityGreaterThanZero": "Quantity must be greater than zero",
-    "QuantityLessThanMax": "Quantity must be smaller",
+    "QuantityInvalid": "Estimated quantity must be a whole number",
+    "QuantityGreaterThanZero": "Estimated quantity must be greater than zero",
+    "QuantityLessThanMax": "Maximum quantity exceeded, enter a lower estimated quantity",
     "PriceRequired": "Enter a price",
     "PriceMustBeANumber": "Price must be a number",
-    "PriceMoreThan3dp": "Price must be to 3 decimal places or less",
+    "PriceMoreThan3dp": "Price must be to a maximum of 3 decimal places",
     "PriceGreaterThanOrEqualToZero": "Price cannot be negative",
-    "PriceLessThanMax": "Price must be smaller",
+    "PriceLessThanMax": "Maximum price exceeded, enter a lower price",
     "DeliveryDateRequired": "Enter a planned delivery date",
     "DeliveryDateInvalid": "Enter a planned delivery date in a valid format",
-    "DeliveryDateOutsideDeliveryWindow": "Planned delivery date must be less than 42 months from the commencement date for this Call-off Agreement",
+    "DeliveryDateOutsideDeliveryWindow": "Planned delivery date must be within 42 months from the commencement date for this Call-off Agreement",
```

Comment:
This 42 month range did change in the API to 183 weeks but not one for this PR.

Reply:
🤷‍♂️ not sure dude. I was given this content. Might be worth just confirming this though